### PR TITLE
feat: Update minter for BSX so royalty can have 2 decimal places

### LINKED ISSUE #4000: Don't see fresh mint in explorer on SNEK
I've made a fresh new mint https://beta.kodadot.xyz/snek/gallery/1559871636-2?page=1&message=congrats&search, and I don't see it in explorer at https://beta.kodadot.xyz/snek/explore?tab=GALLERY&page=1
No errors in the console. 

Is it known bug?

image

plus seems I don't see Creator & Collection on mint
image

### LINKED ISSUE #4005: Update minter for BSX so royalty can have 2 decimal places
### What happened?

Basilisk is now supporting royalties with two decimal places

It would be nice to have it in the minter 

### Expected Behavior

Super Trivial fix:
- extend the slider have smaller steps

Better fix:
- make percent input component some it is user friendly

Bounty applicable for beter fix.

### Ref

- kodadot/snek#65

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

## ⚠️ Caveats

Now the max royalty (100%) is represented on chain as `10_000` (10k)
So to handle it properly you have to multiply the value from to input by `100`

Explainer:
to submit one percent royalty you need to submit 
- 1 `1 * 100 = 100`

### LINKED ISSUE #4025: Cannot load gallery item detail from specific collection /snek
While browsing https://beta.kodadot.xyz/snek/collection/807893860, I cannot load the item detail when I click on any of the items, I get loaded just the placeholder. Happens specifically with this collection, not sure why (most of the others)
image
image

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #4005, closes #4000, closes #4025

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -624,10 +624,10 @@
...
+      "creativeDrop": "Drop your NFT here, click to upload or paste the image from the clipboard. AI model limits our support for Creative Minting to only JPEG, PNG, GIF or BMP image files. The file size must be less than 4MB, and the image's dimensions must be greater than 50x50 pixels. Realistic photographs will achieve the best results.",
       "cover": "Your NFT requires a poster/cover to be seen in gallery. Please upload image (jpg/ png/ gif)",
-      "rmrkDescription": "Generating the title and description failed. Please check whether the uploaded file fits the limits of Creative Minting: JPEG, PNG, GIF or BMP with a size less than 4MB and dimensions greater than 50x50 pixels.",
-      "simpleDrop": "Drop your NFT here or click to upload or simply paste image from clipboard. We support various media types (BMP, GIF, JPEG, PNG, SVG, TIFF, WEBP, MP4, OGV, QUICKTIME, WEBM, GLB, FLAC, MP3, JSON)"
+      "creativeFailDesc": "Generating the title and description failed. Please check whether the uploaded file fits the limits of Creative Minting: JPEG, PNG, GIF or BMP with a size less than 4MB and dimensions greater than 50x50 pixels.",
```
Comment (petersopko) [inline locales/en.json:629]: there was a wrong message regarding the drop of NFT, I've pushed this here since we're fixing minter, ref
- #3886

**Review by petersopko — COMMENTED** [review]
pls check

**Comment by petersopko** [comment]
minted NFT with 10.11 royalty rate no problem, doesn't load on deploy (https://deploy-preview-4008--koda-nuxt.netlify.app/snek/gallery/659233203-1), but after trying to open it on local I get this:
image
doesn't load on deploy (https://deploy-preview-4008--koda-nuxt.netlify.app/snek/gallery/659233203-1)

**Comment by Jarsen136** [comment]
> minted NFT with 10.11 royalty rate no problem, doesn't load on deploy (https://deploy-preview-4008--koda-nuxt.netlify.app/snek/gallery/659233203-1), but after trying to open it on local I get this


Thanks for the test. Already fixed it.
image

**Comment by petersopko** [comment]
> Thanks for the test. Already fixed it. 

in the meantime I found out that this was actually also problem on beta, and not a problem with this specific PR. perhaps related to some of the latest snek changes

**Comment by Jarsen136** [comment]
> perhaps related to some of the latest snek changes

Yes, it's because of the change of decimals of royalty.

**Comment by petersopko** [comment]
setting royalty rate to 0 isn't permitted? 
image

**Comment by Jarsen136** [comment]
> setting royalty rate to 0 isn't permitted? 

Yup, the minimum limit is 0.01%.
Could royalty be 0? I'm not sure about it. @petersopko

**Comment by Jarsen136** [comment]
> > setting royalty rate to 0 isn't permitted?
> 
> Yup, the minimum limit is 0.01%. Could royalty be 0? I'm not sure about it. @petersopko

I found that in the original code, it limits the royalty to be greater than zero.

image

**Review by kkukelka — APPROVED** [review]
code-lgtm, but i'm generally unsure about the hardcoded values here

**Comment by petersopko** [comment]
I'm minting okay, just one last thing regarding the UX, I think it's a little bit of overkill to set the steps to 0.01%, moving up 1% requires me to either type it or tap/click 100 times 😄 can we set it to 0.10% per step?

**Review by petersopko — COMMENTED** [review]
small thing

**Comment by Jarsen136** [comment]
> 703c29d

Good idea! 
✅ Done

**Review by petersopko — COMMENTED** [review]
last small thing

**Comment by Jarsen136** [comment]
> sorry to be a nitpicker but now the user doesn't see there's a possibility to add 0.01% difference (I guess zeros get hidden), can we alter the way we present the numbers there, i.e. always show 0.01% difference (even with zero?) 


How about setting the default value to 0.15% ? 

image

**Review by petersopko — APPROVED** [review]
wfm

**Comment by petersopko** [comment]
> How about setting the default value to 0.15% ?

I think it'll work for now 😄

**Comment by petersopko** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 44.68 USD/KSM ~ 1.119 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x2fb4729f55caeae55b9f6519ccce4f802e5820a1c5dc0200af4ac5d733784498](https://kusama.subscan.io/extrinsic/0x2fb4729f55caeae55b9f6519ccce4f802e5820a1c5dc0200af4ac5d733784498)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)
